beacon-chain/core/validators/validator.go

```diff
@@ -19,6 +20,11 @@ import (
 
 var log = logrus.WithField("prefix", "validator")
 
+type queueElement struct {
+	idx int
+	aEE uint64
```

Comment:
what is queueElement and aEE, can you be more explicit with variable names?

Reply:
changed to ActivationEligibilityEpoch